go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/grafana/terraform-provider-grafana
 
 go 1.17
 
+replace github.com/grafana/grafana-api-golang-client => github.com/grafana/grafana-api-golang-client v0.4.9-0.20220329014453-68ff7155c868
```

Comment:
We should remove this line before merging, right?

Reply:
Yes, definitely will. Once https://github.com/grafana/grafana-api-golang-client/pull/75 is merged